requirements/dev.txt

```diff
@@ -13,3 +13,5 @@ selenium==3.12.0
 djangorestframework==3.8.2
 geopy==1.15.0
 pinax-notifications==5.0.3
+geoip2==3.0.0
+django-ipware==3.0.0
```

Comment:
minor: you can remove this

Reply:
Removing it would cause the code to break. What do you suggest?

Reply:
But this file inherit's from `prod.txt` right, apologies if I am not seeing something.

Reply:
Oh I did not see the first line of dev.txt till date. Sorry